test-framework/junit5/src/main/java/io/quarkus/test/junit/NativeTestExtension.java

```diff
@@ -151,10 +151,15 @@ public void postProcessTestInstance(Object testInstance, ExtensionContext contex
         }
     }
 
-    private void throwBootFailureException() {
+    private void throwBootFailureException() throws Exception {
```

Comment:
Note that the method calling this is already throwing `Exception` so we are perfectly safe doing it here.